src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -3,19 +3,25 @@
 import com.neighborhoodAssurance.dto.AgencyDTO;
 import com.neighborhoodAssurance.service.IAgencyService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.converter.xml.SourceHttpMessageConverter;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestParam;
 
 @Controller
 public class NAController {
-	
+
+	Logger logger = LoggerFactory.getLogger(NAController.class);
+

```

Comment:
I added logging for you, please feel free to use this template + imports throughout your project to debug your issues.

Reply:
This is a good thing to identify/add in a code review.  Are the logging levels appropriate?